style.css

```diff
@@ -148,16 +148,15 @@
   }
 
   .framWorking .html li {
-    color: rgb(34, 30, 30);
-    display: inline-flex;
-    background: #dfe1e6;
-    margin: 10px;
-    position: relative;
-    border-radius: 10px;
-    right: 10%;
+    display: flex;
+    background: #f7f7f9;
+    border-radius: 8px;
+    margin-right: 10%;
+    margin-top: 10px;
```

Comment:
- [ ] In line 9, I can see you used max-width for the mobile version, I would implore you to remove it from the media query, the essence of mobile-first is to style it for the mobile version without using media query and later use a media query to change it to other devices such as the desktop version. The issue with the present way you did it is that any screen width greater than 414px will not appear styled and that is what is affecting it in the attached image.
![image](https://github.com/tchasinga/Setup-first-mobile/assets/86200356/88254cd2-989b-4a45-950d-581573871894)

If you do it based on my recommendation, it may not be perfectly styled when it get to the tab screen size because it is not a requirement but it wont appear not styled as it is appearing presently.